index.html

```diff
@@ -15,22 +15,59 @@
       <!-- T Stewie -->
 
       <div class="title-sun-container">
-        <div class="title">Todo</div>
+        <div class="title">TODO</div>
         <div class="sun-img-container"><img src="images/icon-sun.svg" alt=""></div>
       </div>
 
 
 
 
       <div class="input-container">
-        <input type="text" placeholder="Create a new Todo...">
+        <div class="check-img-container"><img class="empty-img" src="images/icon-check.svg" alt=""></div>
```

Comment:
you can make this a button instead, something like this could work

```html
<div class="input-container">
  <button type="button"></button>
  <input type="text">
</div>
```

Reply:
I am a bit confused on this one, the icon does not seem to be clickable where the user inputs the todo. Making this a button would just mean I would have to deal with extra styling.